### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by stevetoro** [bohemianabe/prj-rev-bwfs-tea-cozy#5]
Needs Improvement

You're on the right track. Make sure to come back and finish this website.

**Comment by bohemianabe** [comment]
Yeah steve this was b.s. (with all due respect) can I have a finished copy so I can look at it myself and learn from it...?!
